composer.lock

```diff
@@ -73,7 +73,7 @@
                 "docblock",
                 "parser"
             ],
-            "time": "2015-08-31 12:32:49"
+            "time": "2015-08-31T12:32:49+00:00"
```

Comment:
@keradus you are right, butt keep in mind that this demo application is a bit special. First, we download it as a ZIP file with all the dependencies installed ... so 99% of users never use Composer with this app. Second, the demo app is designed as a temporary utility (to learn Symfony, to showcase some feature, to do a quick benchmark) so you usually install the demo and delete it pretty quickly ... and then download it again. So 99% of people won't get old dependencies or don't care about them.